gulpfile.js

```diff
@@ -135,8 +135,11 @@ function runJest(...flags) {
 }
 
 const downloadCountryList = async lang => {
-  const res = await superagent.get(`https://raw.githubusercontent.com/umpirsky/country-list/master/data/${lang}/country.json`);
-  await fs.writeFile(`./src/data/country-names/${lang}.json`, JSON.stringify(JSON.parse(res.text), null, 2), 'utf8');
+  const res = await axios.get(
+    `https://raw.githubusercontent.com/umpirsky/country-list/master/data/${lang}/country.json`,
```

Comment:
I know that it was that sloppy before but with this change we would have the chance to correct all URL interpolations with proper escaping like `https://raw.githubusercontent.com/umpirsky/country-list/master/data/${encodeURIComponent(lang)}/country.json`. I'll put a '+1' on each place I see and leave the decision to you (in a basically all cases it's more a theoretical matter as we control the params there)

Reply:
Yeah very good point, let's do that then!
Thanks for the +1 to help find them, will change.

Reply:
-1